src/Repository/QuizInstanceRepository.php

```diff
@@ -21,7 +21,7 @@ public function getQuestionsNumber(int $quizTemplateId): int
 
     public function getResultsData(): array
     {
-        $sql = "SELECT quiz_instance.id AS quiz_instance_id, user.name, quiz_template.id AS quiz_template_id FROM quiz_instance 
+        $sql = "SELECT quiz_instance.id AS quiz_instance_id, user.name, quiz_template.id AS quiz_template_id, user.id AS user_id FROM quiz_instance 
```

Comment:
These selects are very raw. Could you for example use a template method to return actual table name and implement it in every repository, also have an easy way to get another's entity main table by calling a method on its repository. I know  you weren't suppose to build your own DBAL, but at least try to make you life easier by automating some of the query building.